frontend/src/routes/Infrastructure/Clusters/ManagedClusters/CreateCluster/components/AvailabilityOptionsForm.test.tsx

```diff
@@ -0,0 +1,64 @@
+/* Copyright Contributors to the Open Cluster Management project */
+import { render, screen } from '@testing-library/react'
+import { RecoilRoot } from 'recoil'
+import { MemoryRouter, Route, Routes } from 'react-router-dom-v5-compat'
+import { NavigationPath } from '../../../../../../NavigationPath'
+
+import AvailabilityOptionsForm from './AvailabilityOptionsForm'
+import { waitForText } from '../../../../../../lib/test-util'
+import userEvent from '@testing-library/user-event'
+
+describe('Availability Options Form', () => {
+  const handleChange = jest.fn()
+  const activeControl = {
+    controller: 'HighlyAvailable',
+    infra: 'HighlyAvailable',
+  }
+
+  const Component = () => {
+    return (
+      <RecoilRoot>
+        <MemoryRouter initialEntries={[NavigationPath.createCluster]}>
+          <Routes>
+            <Route
+              path={NavigationPath.createCluster}
+              element={
+                <AvailabilityOptionsForm
+                  control={{
+                    active: activeControl,
+                  }}
+                  handleChange={handleChange}
+                />
+              }
+            />
+          </Routes>
+        </MemoryRouter>
+      </RecoilRoot>
+    )
+  }
+
+  test('renders', async () => {
+    const { container } = render(<Component />)
+    await waitForText('Controller availability policy')
+    expect(container).toMatchSnapshot()
+  })
```

Comment:
I expect this is not needed for coverage (since the next test renders the same thing), so I don't think we need a snapshot test here. Can we remove?